homeassistant/components/airvisual/sensor.py

```diff
@@ -315,29 +315,39 @@ def device_info(self) -> DeviceInfo:
             ),
         }
 
+    def unique_id(self):
```

Comment:
Why? The unique ID is already defined via `self._attr_unique_id`.

Reply:
Thanks @bachya, fixed in 2c1d80e